docs/src/markdown/installation.md

```diff
@@ -22,10 +22,15 @@ Name                             | Details
     ```
 
     Replace `python2.7-dev` with the Python version you are using.
+    
+    For Fedora 26, you need few dependecies to built wxPython:
```

Comment:
Please have one blank line before code blocks.

Reply:
Done

Reply:
Don't forget to fix the spelling of "dependencies".  Spelling test is failing because of it.

Reply:
@facelessuser yeah, I just notice about it.